wiki/Tournaments/The_Roundtable/en.md

```diff
@@ -28,14 +28,19 @@ tags:
 | :-- | :-- |
 | Host | ::{ flag=US }:: [BTMC](https://osu.ppy.sh/users/3171691) |
 | Project Manager | ::{ flag=US }:: [Kroytz](https://osu.ppy.sh/users/2339768), ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) |
-| Referee | ::{ flag=US }:: [Tekkito](https://osu.ppy.sh/users/7075211), ::{ flag=US }:: [D I O](https://osu.ppy.sh/users/3958619), ::{ flag=US }:: [freddiiieeee](https://osu.ppy.sh/users/7112839) |
-| Production | ::{ flag=RU }:: [DeadRote](https://osu.ppy.sh/users/7943969), ::{ flag=GB }:: [Jame](https://osu.ppy.sh/users/14167692), Albert Chang |
+| Commentators | ::{ flag=US }:: [BTMC](https://osu.ppy.sh/users/3171691) ::{ flag=US }:: [HappyStick](https://osu.ppy.sh/users/256802) |
+| Referee | ::{ flag=US }:: [Tekkito](https://osu.ppy.sh/users/7075211), ::{ flag=US }:: [D I O](https://osu.ppy.sh/users/3958619), ::{ flag=US }:: [freddiiieeee](https://osu.ppy.sh/users/7112839), ::{ flag=US }:: [\[K\]](https://osu.ppy.sh/users/16551387), ::{ flag=GB }:: [ilw8](https://osu.ppy.sh/users/14167692), ::{ flag=GB }:: [Yazzeh](https://osu.ppy.sh/users/14834150) |
+| Production | ::{ flag=RU }:: [DeadRote](https://osu.ppy.sh/users/7943969), ::{ flag=GB }:: [ilw8](https://osu.ppy.sh/users/14167692), ::{ flag=US }:: Albert Chang |
 | Designer | ::{ flag=US }:: [Flezlin](https://osu.ppy.sh/users/3696423), ::{ flag=US }:: [joiechii](https://osu.ppy.sh/users/9995116) |
-| Map Pooler | ::{ flag=US }:: [Tekkito](https://osu.ppy.sh/users/7075211), ::{ flag=US }:: [D I O](https://osu.ppy.sh/users/3958619), ::{ flag=FR }:: [PLAYOSUDROID](https://osu.ppy.sh/users/8642284), ::{ flag=US }:: [fum0](https://osu.ppy.sh/users/3715823) ::{ flag=FR }:: [Ekoro](https://osu.ppy.sh/users/284905) |
-| Map Pool Playtester | ::{ flag=PH }:: [BronyPH](https://osu.ppy.sh/users/1492995) |
+| Map Pooler | ::{ flag=US }:: [Tekkito](https://osu.ppy.sh/users/7075211), ::{ flag=US }:: [D I O](https://osu.ppy.sh/users/3958619), ::{ flag=FR }:: [PLAYOSUDROID](https://osu.ppy.sh/users/8642284), ::{ flag=US }:: [ChaosRaidz](https://osu.ppy.sh/users/3715823), ::{ flag=FR }:: [Ekoro](https://osu.ppy.sh/users/284905), ::{ flag=PH }:: [BronyPH](https://osu.ppy.sh/users/1492995) |
```

Comment:
```suggestion
| Map Pooler | ::{ flag=US }:: [Tekkito](https://osu.ppy.sh/users/7075211), ::{ flag=US }:: [D I O](https://osu.ppy.sh/users/3958619), ::{ flag=FR }:: [Nerowo](https://osu.ppy.sh/users/8642284), ::{ flag=US }:: [ChaosRaidz](https://osu.ppy.sh/users/3715823), ::{ flag=FR }:: [Ekoro](https://osu.ppy.sh/users/284905), ::{ flag=PH }:: [BronyPH](https://osu.ppy.sh/users/1492995) |
```

There is no player associated under `PLAYOSUDROID` unless they are restricted or something.

Reply:
looks like they changed name oops